project1/index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+
+
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Caesar+Dressing|Metal+Mania|Nunito+Sans|Squada+One&display=swap"
+        rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans+Caption:700&display=swap" rel="stylesheet">
+
+</head>
+
+
+<body>
+    <main>
+        <div class="cut">
+            <h1><span class="Hlight">The Crane Wife</span></h1>
+            <h2>By CJ Hauser</h2>
+            <h3>July 16, 2019</h3>
+        </div>
+
+        <p><span class="first">Ten days after</span> I called off my engagement I was supposed to go on a scientific
+            expedition to study the
+            whooping
+            crane on the gulf coast of Texas. Surely, I will cancel this trip, I thought, as I shopped for nylon
+            hiking
+            pants that zipped off at the knee. Surely, a person who calls off a wedding is meant to be sitting sadly
+            at
+            home, reflecting on the enormity of what has transpired and not doing whatever it is I am about to be
+            doing that
+            requires a pair of plastic clogs with drainage holes. Surely, I thought, as I tried on a very large and
+            floppy
+            hat featuring a pull cord that fastened beneath my chin, it would be wrong to even be wearing a hat that
+            looks
+            like this when something in my life has gone so terribly wrong.</p>
+
+        <p>
+            <b class="first">Ten days <span class="letter">earlier</span></b> I had cried and I had yelled and I had
```

Comment:
The "letter" class and the `span` here aren't doing anything different than the "first" class and the `b element?
```suggestion
            <b class="first">Ten days earlier</b> I had cried and I had yelled and I had
```